HotelReservation/HotelReservationTests/Hotels/HotelSystemTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using HotelReservation.Hotels;
+using HotelReservation.ReservationSteps;
+using NUnit.Framework;
+
+namespace HotelReservationTests.Hotels
+{
+    [TestFixture]
+    public class HotelSystemTests
```

Comment:
Brakuje testów na:
* LastHotelId - a to publiczny interfejs. Ta propercja wg mnie jest niepotrzebna ale w momencie jej dodawania powinien powstać test
* GetHotelReservationSteps kiedy podany Id nie istnieje

Reply:
Fixed

Reply:
ShouldLastHotelIdBe0WhenNoHotelsAdded
ID jest numerem - 0 to też numer. Taka knonwencja jest bardzo niebezpieczna i prędzej czy później w projekcie gdzie jest kilka osób ktoś przeoczy tego typu zagrania. Bezpieczniej będzie nie zwrócić id jeśli nie istnieje

Reply:
Fixed